scripts/setup.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+echo "alias docker-compose='docker compose'" >> ~/.bashrc
```

Comment:
If someone already has `docker-compose` installed but not `docker compose` will this work?